ledger-tool/src/main.rs

```diff
@@ -4185,55 +4190,104 @@ fn main() {
                     &shred_storage_type,
                     force_update_to_open,
                 );
+
                 match blockstore.slot_meta_iterator(0) {
                     Ok(metas) => {
-                        let all = arg_matches.is_present("all");
+                        let output_format =
+                            OutputFormat::from_matches(arg_matches, "output_format", false);
+                        let all = matches.is_present("all");
 
                         let slots: Vec<_> = metas.map(|(slot, _)| slot).collect();
+                        let mut slot_bounds = SlotBounds::default();
+
                         if slots.is_empty() {
-                            println!("Ledger is empty");
+                            match output_format {
+                                OutputFormat::Json | OutputFormat::JsonCompact => {
+                                    println!("{}", output_format.formatted_string(&slot_bounds))
+                                }
+                                _ => println!("Ledger is empty"),
+                            }
                         } else {
                             let first = slots.first().unwrap();
                             let last = slots.last().unwrap_or(first);
-                            if first != last {
-                                println!(
-                                    "Ledger has data for {} slots {:?} to {:?}",
-                                    slots.len(),
-                                    first,
-                                    last
-                                );
-                                if all {
-                                    println!("Non-empty slots: {slots:?}");
+
+                            slot_bounds.total_slots = Some(SlotInfo {
+                                start: *first,
+                                end: *last,
```

Comment:
"Start" and "end" are not intuitive for me. How about "first/last" or "lowest/highest"?

Reply:
First/last are great, updating 👍🏻